### PR TITLE
Add RDS Civilian Pack detection

### PR DESCRIPTION
**Pull request by azgaviperr**
## What type of PR is this.
1. [ ] Bug
2. [X] Enhancement

### What have you changed and why?
Information: This change is an enhancement of the RHS vibes by introducing the RDS Civilian Pack mod to replace futuristic vehicles which break the feeling of the Cold War introduced byt RHS (Specially the SUV). 

Mod used is here: https://steamcommunity.com/sharedfiles/filedetails/?id=612930542
alt text

    
### Please specify which Issue this PR Resolves.
closes nothing

### Please verify the following and ensure all checks are completed.

1. [X] Have you loaded the Mission in Singleplayer?
2. [X] Have you loaded the Mission in a Dedicated Server?

### Is further testing or are further changes required?
1. [ ] No
2. [X] Yes (Please provide further detail below.)

********************************************************
Notes:

Spawning rate may need to be arranged, but it is working as it is. 
Old bike does explose while higly damage ^^

### CODE REVIEW

File: A3-Antistasi/functions/init/fn_initVarServer.sqf [hunk 1]
```diff
@@ -464,6 +465,8 @@ if !(hasIFA) then {
 	switch (true) do {
 		case (!activeAFRF): {call compile preProcessFileLineNumbers "Templates\Vanilla_Civ.sqf";};
 		case (has3CB): {call compile preProcessFileLineNumbers "Templates\3CB_Civ.sqf"};
+		case (hasRDS): {call compile preProcessFileLineNumbers "Templates\RDS_Civ.sqf"};
+		case (activeAFRF): {call compile preProcessFileLineNumbers "Templates\RHS_Civ.sqf"};
```
Comment (Lazejun) [inline A3-Antistasi/functions/init/fn_initVarServer.sqf:469]: Once AFRF should be enough :D

File: A3-Antistasi/Templates/RDS_Civ.sqf [hunk 2]
```diff
@@ -0,0 +1,46 @@
+
+civVehCommonData = [
+	 "C_Quadbike_01_F", 0.3
+	,"RDS_Old_bike_Civ_01", 2.0 // Bike ca nexplose as it is full of Vodka
```
Comment (Lazejun) [inline A3-Antistasi/Templates/RDS_Civ.sqf:4]: I get that its funny but there isnt realy a point of having a Bike thats pretty useless on 2.0

**Comment by jaj22** [comment]
It looks like you're merging a lot of PR branches into this, which makes it difficult to handle. Recommended way to do things:

1. Have your "unstable" branch track official-antistasi-community:unstable. Don't modify it directly.
2. Make your local modifications on new branches based on this unstable branch.
3. Create pull requests on these new branches.
4. If you want to merge other PRs for local use, make a branch specifically for that.

**Comment by azgaviperr** [comment]
> 
> 
> It looks like you're merging a lot of PR branches into this, which makes it difficult to handle. Recommended way to do things:
> 
>     1. Have your "unstable" branch track official-antistasi-community:unstable. Don't modify it directly.
> 
>     2. Make your local modifications on new branches based on this unstable branch.
> 
>     3. Create pull requests on these new branches.
> 
>     4. If you want to merge other PRs for local use, make a branch specifically for that.

Yeah, I was on the wrong branch on my side when I did commit/push . Sorry for that. 

Do you need me to redo the PR ?

**Comment by azgaviperr** [comment]
New pull request made with proper branch

**Pull request closed without merging**